lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/LibertyDiagnosticParticipant.java

```diff
@@ -101,52 +104,68 @@ private void validateDom(DOMDocument domDocument, List<Diagnostic> diagnosticsLi
         validateConfigElements(domDocument, diagnosticsList, tempDiagnosticsList, featureGraph, includedFeatures, featureManagerPresent);
     }
 
-    private void validateFeature(DOMDocument domDocument, List<Diagnostic> list, DOMNode featureManager, Set<String> includedFeatures) {
+    private void validFeaturesAndPlatforms(DOMDocument domDocument, List<Diagnostic> list, DOMNode featureManager, Set<String> includedFeatures) {
```

Comment:
I mistyped in my previous comment. Should still say validate....`validateFeaturesAndPlatforms`.